docs/content/markdown.fsx

```diff
@@ -64,7 +67,7 @@ The following snippet prints the heading of the document:
 // Iterate over all the paragraph elements
 for par in parsed.Paragraphs do
   match par with
-  | Heading(1, [Literal text]) -> 
+  | Heading(1, [Literal(text, _)], _) -> 
```

Comment:
To encourage people writing the code in the new style - and future-proof way, could you change the pattern matching here to use the naming? For example:

    | Heading(size=1; body=[Literal(text=text)]) -> 

I know this is quite tedious (!) but adding the names to the DUs is a fantastic improvement for the library and it would be great to have it in a couple of places in the docs, so that people know about it - and use it!